src/MeadeCommandProcessor.cpp

```diff
@@ -816,6 +816,22 @@ bool gpsAqcuisitionComplete(int &indicator);  // defined in c72_menuHA_GPS.hpp
 //      Returns:
 //        "HHMMSS"
 //
+// :XGh#
+//      Description:
+//        Get Heater values
+//      Information:
+//        Get the current values for both heaters
+//      Returns:
+//        "NNN,NNN"
```

Comment:
Can we make the get and set functions symmetrical, i.e. have the get also use a N to signify which heater? You can make the number optional to save a call. So support both `:XGh#` and `:XGh1#`.

Reply:
I actually did it that way because you had it like that for the XDGL...

Reply:
I know, but I have realized this is not optimal :-) But supporting both is straightforward.

Reply:
So we should be good now.

Reply:
I don't see the optional character documented....

Reply:
//        Get value for heater N (0 or 1) as a value in the range from 0-10, or both heaters if no N is given.